src/main/resources/META-INF/plugin.xml

```diff
@@ -377,7 +377,12 @@ L
         <!-- LSP textDocument/selectionRange request support -->
         <extendWordSelectionHandler
                 implementation="com.redhat.devtools.lsp4ij.features.selectionRange.LSPExtendWordSelectionHandler"
+                id="LSPExtendWordSelectionHandler"
                 order="first"/>
+        <extendWordSelectionHandler
+                implementation="com.redhat.devtools.lsp4ij.features.selectionRange.LSPCodeBlockExtendWordSelectionHandler"
```

Comment:
It's important that the "wider" `extendWordSelectionHandler` be registered before the "narrower" one.